qiskit_addon_sqd/fermion.py

```diff
@@ -593,3 +610,17 @@ def _transition_str_to_bool(string_rep: np.ndarray) -> tuple[np.ndarray, np.ndar
     annihilate = np.logical_or(string_rep == "-", string_rep == "n")
 
     return diag, create, annihilate
+
+def generate_bitstrings(norb, neleca,nelecb, n_orb, n_alpha, n_beta):
+    def get_strings(n, k):
+        return [''.join('1' if i in comb else '0' for i in range(n))
+                for comb in combinations(range(n), k)]
+    alpha_strings = get_strings(n_orb, n_alpha)
+    beta_strings = get_strings(n_orb, n_beta)
+    d_orb = neleca+nelecb - n_alpha - n_beta
+    doubly_occ = [''.join(seq) for seq in itertools.product('1', repeat=int(d_orb/2))]
```

Comment:
This line is equivalent to
```python
double_occ = ['1' * int(d_orb / 2)]
```
Is this what you meant to write? I'm having a bit of trouble following the logic.